client-app/src/desktop/tabs/features/NewsPanel.js

```diff
@@ -0,0 +1,106 @@
+import {HoistComponent} from '@xh/hoist/core/index';
+import {NewsPanelModel} from "./NewsPanelModel";
+import {dataView} from '@xh/hoist/desktop/cmp/dataview';
+import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {wrapper} from '../../common/Wrapper';
+import {filler, span} from "@xh/hoist/cmp/layout";
+import {Icon} from "@xh/hoist/icon";
+import {toolbar} from "@xh/hoist/desktop/cmp/toolbar";
+import {multiSelectField} from "@xh/hoist/desktop/cmp/form";
+import {storeFilterField} from "@xh/hoist/desktop/cmp/store"
+import {button} from "@xh/hoist/desktop/cmp/button";
+import {storeCountLabel} from '@xh/hoist/desktop/cmp/store';
+import {relativeTimestamp} from "@xh/hoist/cmp/relativetimestamp"
+import './NewsPanelItem.scss';
+
+
+
+
+@HoistComponent
+export class NewsPanel extends Component {
+
+    localModel = new NewsPanelModel();
+
+
+    render() {
+        const {model} = this,
+            {viewModel} = model;
+
+        return wrapper({
+            item: panel({
+                className: 'toolbox-news-panel',
+                title: 'News Feed',
+                width: '75%',
+                height: '90%',
+                item: dataView({
+                    model: viewModel,
+                    rowCls: 'news-item',
+                    itemHeight: 120,
+                    onRowDoubleClicked: this.onRowDoubleClicked
+                }),
+                tbar: toolbar({
+                    items: [
+                        button({
+                            text: 'Refresh Sources',
+                            icon: Icon.refresh(),
+                            onClick: this.onRefreshClick
+                        }),
+                        filler(),
+                        span({
+                            item: "Last updated:"
+                        }),
+                        relativeTimestamp({
+                            timestamp: model.lastRefresh
+                        })
+
+                    ]
+                }),
+                bbar: toolbar({
+                    items: [
+                        storeFilterField({
+                            onFilterChange: this.onFilterChange,
+                            fields: model.SEARCH_FIELDS,
+                            placeholder: "Select by title"
+                        }),
+                        multiSelectField({
+                            placeholder: "Filter sources...",
+                            options: model.sourceOptions,
+                            commitOnChange: true,
+                            model,
+                            field: 'sourceFilter'
+                        }),
+                        filler(),
+                        storeCountLabel({
+                            store: viewModel.store,
+                            unit: 'stories'
+                        })
+                    ]
+                })
+            })
+        })
+    }
+
+    componentDidMount() {
+        this.model.loadData();
```

Comment:
define loadAsync() here instead.  This is the root of a Tab.  This will also allow that the tab to play well with global refreshing of its parent container.